code/modules/events/event_manager.dm

```diff
@@ -34,19 +34,22 @@ var/scheduledEvent = null
 			if(0 to 18000) //30 first minutes
 				roundlength_modifier = 1 //Don't particularly speed up event rate
 			if(18000 to 36000) //Next thirty minutes
-				roundlength_modifier = 0.95 //Start ramping up
+				roundlength_modifier = 0.7 //Start ramping up
 			if(36000 to 54000)
-				roundlength_modifier = 0.90
+				roundlength_modifier = 0.6
 			if(54000 to INFINITY) //Round has been going for 1 hour and 30 minutes at least
-				roundlength_modifier = 0.85
```

Comment:
you don't mention these tweaks at all in your changelog

Reply:
time, independent of playercount:
old; new
9.5,19; 14,28
9,18; 12,24
8.5,17; 8,16